src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -303,3 +303,57 @@
             [(conj errors {:index row-index, :error (ex-message e)})
              created-rows]))))
      rows)))
+
+(defn- pk-values->filters [table-id pk-values]
```

Comment:
I would consider naming this to something like 

```
pk-value-maps->honeysql-filter-clauses
```

to clarify it a bit. 

`pk-values` is a bit ambiguous -- is it a single map? A single vector? `pk-values-maps` makes it clear that it's a sequence of maps

`filters` is a little ambiguous too -- is it a sequence of filtering predicate functions? A sequence of MBQL `:filter` clauses? `honeysql-filter-clauses` removes the ambiguity.

When in doubt, it's definitely better to error on the side of making your function and variable names too verbose. Ideally a function name alone should be enough to tell you what it does with ambiguity. If that's not feasible, a docstring with an example usage and return value goes a long way.